### PR TITLE
Added option to make toast undismissible

### PR DESCRIPTION
**Pull request by GermainBergeron**
Added example
Added unit tests

undimissible-toast

### CODE REVIEW

**Comment by MichelML** [comment]
In what cases would you want such a behavior @GermainBergeron ?

**Comment by GermainBergeron** [comment]
In some cases the error should be visible until the user fix the problem.

In my specific use case a user can add many cards to a dashboard and go over the size limit. To fix his problem he can resize a card, remove a card, etc. Until he fixes the situation the toast will stay visible (and he won't be able to save)

File: src/components/toast/examples/ToastExamples.tsx [hunk 1]
```diff
@@ -56,6 +59,8 @@ export class ToastExamples extends React.Component<{}, IToastExamplesState> {
               </Toast>}
               {this.state.Timed &&
                 <Toast key='toast-4' id='toast-4' title='Timed Success!' dismiss={1000} onClose={() => this.setState({ Timed: false })} />}
+              {this.state.NonDismissible &&
+                <Toast key='toast-5' id='toast-5' title='Ohno! Something went wrong.' type={ToastType.Error} dismissible={false} content={ToastContentExample} />}
```
Comment (MichelML) [inline src/components/toast/examples/ToastExamples.tsx]: almost shy to make this comment but `Ohno!` should be `Oh no!` ....

File: src/components/toast/Toast.tsx [hunk 2]
```diff
@@ -45,7 +50,7 @@ export class Toast extends React.Component<IToastProps, {}> {
   }
 
   private setTimeout() {
-    if (this.props.dismiss > 0) {
+    if (this.props.dismissible !== false && this.props.dismiss > 0) {
```
Comment (sroy3) [inline src/components/toast/Toast.tsx]: You don't need `!== false`

**Pull request merged**